source/common/router/config_impl.cc

```diff
@@ -378,6 +378,10 @@ CorsPolicyImpl::CorsPolicyImpl(const envoy::config::route::v3::CorsPolicy& confi
   if (config.has_allow_credentials()) {
     allow_credentials_ = PROTOBUF_GET_WRAPPED_REQUIRED(config, allow_credentials);
   }
+  if (config.has_allow_private_network_access()) {
+    allow_private_network_access_ =
+        PROTOBUF_GET_WRAPPED_REQUIRED(config, allow_private_network_access);
+  }
```

Comment:
What's the benefit of using this over `PROTOBUF_GET_WRAPPED_OR_DEFAULT` without an optional value?

Reply:
Thanks for the suggestion, use `PROTOBUF_GET_WRAPPED_OR_DEFAULT` is better here.